### DISCUSSION TITLE
PBTaskAddEditor 2.0 creo que falla cuando se utiliza tipo accessrole

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#307] [4. Bug Reports] [answered]
Hola

Quiero agregar usuarios a una tarea mediante el bot PBTaskAddEditor
De muchos intentos dejé el bot de esta forma como se ve en la base de datos:

{
                "_id" : ObjectId("60dc7df5cde2870008632042"), 
                "key" : "invite_hepatico", 
                "name" : "PBTaskAddEditor", 
                "version" : "2.0.0", 
                "data" : {
                    "type" : "accessRole", 
                    "accessRole" : [
                        "6054f45a6cbbf4000889c154"
                    ], 
                    "task" : "$OUTPUT#crear_paciente_hepatico#task|_id", 
                    "taskGroup" : "6054f4e16aca6000088a355e"
                }, 
                "next" : {
                    "SUCCESS" : "send_hepatico", 
                    "ERROR" : ""
                }, 
                "customNetworkRequest" : [

                ]
            },
**El resultado me arrojó esto:**
![image](https://user-images.githubusercontent.com/61419471/123983984-4752e680-d992-11eb-92a4-4c5101cab030.png)

**Sin embargo el rol existe**
![image](https://user-images.githubusercontent.com/61419471/123984059-53d73f00-d992-11eb-9516-1a8e28e94008.png)

Gracias=>

**Comment by guillevalin** [comment, marked as answer]
@NicolasPLG Se subió una corrección dado que accessRole es el nombre, en este caso "hepatico" y no el ID del accessRole.

Saludos.

**Reply by NicolasPLG** [reply]
gracias guille, esperaré la corrección para actualzar el bot
